docs/ReleaseNotes.md

```diff
@@ -70,7 +72,7 @@ Methods for retrieving a record from a record store based on an index entry gene
 * **Breaking change** `IndexMaintainer`s should implement `validateEntries` to validate orphan index entries [(Issue #383)](https://github.com/FoundationDB/fdb-record-layer/issues/383)
 * **Breaking change** Change 4 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Breaking change** The API stability annotations have been moved into `com.apple.foundationdb.annotation` [(Issue #406)](https://github.com/FoundationDB/fdb-record-layer/issues/406)
-
+* **Breaking change** `SubspaceProvider`receives `FDBRecordContext` when a subspace is resolved vs. at contstruction.[(Issue #338)](https://github.com/FoundationDB/fdb-record-layer/issues/338)
```

Comment:
```suggestion
* **Breaking change** `SubspaceProvider` receives an `FDBRecordContext` when a subspace is resolved instead of at contstruction [(Issue #338)](https://github.com/FoundationDB/fdb-record-layer/issues/338)
```

This should probably retain the blank line before the `// end next release` line

Reply:
It looks like only some of this suggestion (turning "vs." to "instead of") was applied in the latest commit but not the rest.